cluster/gce/config-test.sh

```diff
@@ -317,7 +319,7 @@ fi
 # IP_ALIAS_SIZE is the size of the podCIDR allocated to a node.
 # IP_ALIAS_SUBNETWORK is the subnetwork to allocate from. If empty, a
 #   new subnetwork will be created for the cluster.
-ENABLE_IP_ALIASES=${KUBE_GCE_ENABLE_IP_ALIASES:-false}
```

Comment:
LGTM.

In the future, we should change our test job to set `KUBE_GCE_ENABLE_IP_ALIASES` and keep the default `false`.